k8s/autopilot/autopilotruleobject.go

```diff
@@ -0,0 +1,59 @@
+package autopilot
+
+import (
+	autv1alpha1 "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RuleObjectOps is an interface to perform k8s AutopilotRuleObjects operations
+type RuleObjectOps interface {
+	// CreateAutopilotRuleObject creates the AutopilotRuleObject object
+	CreateAutopilotRuleObject(namespace string, object *autv1alpha1.AutopilotRuleObject) (*autv1alpha1.AutopilotRuleObject, error)
```

Comment:
remove `namespace` from CreateAutopilotRuleObject. It just should just the namespace set in the provided object.

Reply:
fixed